### PR TITLE
Update image metadata to use LABEL

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

**Comment by mfojtik** [comment]
https://github.com/openshift/sti-ruby/issues/24

**Comment by soltysh** [comment]
[test]

File: 3.3/.sti/bin/run [hunk 1]
```diff
@@ -19,7 +19,7 @@ if is_gunicorn_installed && [[ -v APP_MODULE ]]; then
     if [[ -v APP_CONFIG ]]; then
         export CONFIG="--config ${APP_CONFIG}"
     fi
-    exec gunicorn ${APP_MODULE} --bind=:${IMAGE_EXPOSE_SERVICES%:*} ${CONFIG}
+    exec gunicorn ${APP_MODULE} --bind=8080 ${CONFIG}
```
Comment (soltysh) [inline 3.3/.sti/bin/run]: Here's the bug you need to prepend colon ':' before 8080 port.

**Comment by soltysh** [comment]
re-[test] as previous error was related to timeouts on scl site, which appears to be working correctly now.

**Comment by soltysh** [comment]
LGTM

**Pull request merged**